src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerImpl.java

```diff
@@ -114,7 +114,7 @@ private LiKafkaConsumerImpl(LiKafkaConsumerConfig configs,
     _lastAutoCommitMs = System.currentTimeMillis();
     // We need to set the auto commit to false in KafkaConsumer because it is not large message aware.
     ByteArrayDeserializer byteArrayDeserializer = new ByteArrayDeserializer();
-    _kafkaConsumer = new KafkaConsumer<>(configs.configForVanillaConsumer(),
+    _kafkaConsumer = new KafkaConsumer<>(configs.configForVanillaConsumer().originals(),
```

Comment:
In LiKafkaConsumerConfig we wrapped the map into a `LiKafkaConsumerConfig`, and here we unwrap it. Is there any special consideration?